volume/mounts/windows_parser.go

```diff
@@ -47,7 +47,7 @@ const (
 	rxName = `[^\\/:*?"<>|\r\n]+`
 
 	// RXReservedNames are reserved names not possible on Windows
-	rxReservedNames = `(con)|(prn)|(nul)|(aux)|(com[1-9])|(lpt[1-9])`
+	rxReservedNames = `((con)|(prn)|(nul)|(aux)|(com[1-9])|(lpt[1-9]))`
```

Comment:
Looking at this regex, I think this could even be simplified to:

```suggestion
	rxReservedNames = `(con|prn|nul|aux|com[1-9]|lpt[1-9])`
```

Reply:
Yeah, you're right. just updated.

Reply:
At that point it doesn't even need the outer capturing, right?



```suggestion
	rxReservedNames = `con|prn|nul|aux|com[1-9]|lpt[1-9]`
```

Reply:
It does need them - that was the original bug.

"^con|prn|nul|aux|com[1-9]|lpt[1-9]$" matches "test-aux-volume"

"^(con|prn|nul|aux|com[1-9]|lpt[1-9])$" does not